compute/disks/disks_test.go

```diff
@@ -465,4 +465,40 @@ func TestComputeDisksSnippets(t *testing.T) {
 		// Cannot clean up the disk just yet because it must be done after the VM is terminated.
 		// It will be done by deleteInstance function.
 	})
+	t.Run("Create a test disk and list it", func(t *testing.T) {
```

Comment:
nit: this test does not create a disk. a more accurate comment would be "list disks"

Reply:
fixed